src/main/java/org/isf/supplier/gui/SupplierBrowser.java

```diff
@@ -337,7 +337,7 @@ public Object getValueAt(int r, int c) {
 			} else if (c == 7) {
 				return sup.getSupNote();
 			} else if (c == 8) {
-				return sup.getSupDeleted().equals('Y');
+				return sup.getActive().equals(0);
```

Comment:
Doesn't `getActive()` return an `int`?   So why isn't the test `return sup.getActive() == 1`?